client/src/components/admin/Notifications/NotificationForm.vue

```diff
@@ -99,9 +86,37 @@ async function loadData<T>(
     try {
         const tmp = await getData();
         target.value = tmp.map(formatter);
-    } catch (error: any) {
+    } catch (error) {
+        Toast.error(errorMessageAsString(error));
```

Comment:
I think you can remove this since you're not throwing in any of the functions passed in as getData ?